README.md

```diff
@@ -1652,16 +1664,16 @@ These scripts use that value to create a new environment variable **`COMPOSITE_2
 * Load global environment variables defined in the **env.\[current profile\]** block.
 * Load global environment setup scripts defined in the **env_scripts** attribute.
 * **Per Task**
-  * Load environment files defined in the **env_files** attribute (relative paths are treated differently than global env_files).
   * Setup **per task** internal environment variables (see [Global](#usage-env-global) section).
+  * Load environment files defined in the **env_files** attribute (relative paths are treated differently than global env_files).
   * Load environment variables defined in the **env** block (same behavior as global env block).
```

Comment:
@sagiegurari This looks like the existing documentation was wrong, so I've corrected it to match what the code actually does (i.e. internal are expanded before env_files